.github/workflows/.ansible-test.yml

```diff
@@ -32,11 +28,14 @@ jobs:
           if [ -f test_requirements.txt ]; then pip install -r test_requirements.txt; fi
```

Comment:
i think this should in tests/ test_requirements.txt

Reply:
if [ -f tests/test_requirements.txt ]; then pip install -r tests/test_requirements.txt; fi

Reply:
Thanks for pointing this. Fixed it.